docs/governance/templates/release_issue.md

```diff
@@ -58,8 +58,8 @@ and copy it into a release issue. Fill in relevant values, found inside {}
 - [ ] Attach all assets found in the cloud storage with {version} to the draft GitHub Release.
 - [ ] Copy any review changes from the release blog post into the draft GitHub release.
 - [ ] Publish the draft GitHub Release.
-- [ ] Run `make release-branch` to create a release branch.
-- [ ] Email the [mailing list][list] with the release details (copy-paste the release blog post).
+- [ ] Run `make release-branch` to create a release branch and run `gcloud config configurations activate <your development project>` to switch Agones development tooling off of the `agones-images` project.
+- [ ] Email the `mailing list` with the release details (copy-paste the release blog post). `Internal Agones Release Guide` has the mailing list.
```

Comment:
nit: Not sure whether it matters but `Internal Agones Release Guide` here is not very clear to those who don't know about the doc. Maybe explicitly call out the mailing lists here.

Reply:
If my understanding is correct, we should refrain from sharing the internal document in a public forum. @markmandel, please correct me if I'm mistaken.

Reply:
We can link to the public google group - that's fine. Maybe something like the following:

```suggestion
- [ ] Email all mailing lists with the release details (copy-paste the release blog post). 
  - [ ] Public (agones discuss)[list] mailing list
  - [ ] Googlers: Check the [Internal Release Guide](go/agones-release-guide) for extra steps at this stage.
```

We use go links in a few projects to link to internal stuff - but also being deliberately vague 😄

Reply:
I have included both the public email list and the Googlers email list in the [Internal] Agones Release Guide and added the corresponding link(go/agones-release-guide). Does it look good?